Reportedly/Modules/Main/Home/HomeViewController.swift

```diff
@@ -353,12 +368,20 @@ extension HomeViewController {
         guard rightMenuIsShowing != shouldShow else { return }
         
         rightMenuIsShowing = shouldShow
-        UIView.animate(withDuration: 0.5, delay: 0.0, usingSpringWithDamping: 4.0, initialSpringVelocity: 0.8, options: .curveEaseInOut) { [weak self] in
+        
+        if shouldShow {
```

Comment:
I think we should have a function to update `menuDismissRegionView ` from `presenter` like `set MenuDismissRegionViewHidden ` or something like that. What do you think?

Reply:
hmm `menuDismissRegionView` works together with `menuContainerView` to create a dark layer effect when the `menuBar` is showing, and we already have the function `hideRightMenuBar` inputs from `presenter` already, we don't need to hide `menuDismissRegionView` separately 🙏

Reply:
my bad, i didn't read the code carefully 😅 . Please ignore it 🙏